app/views/get-started/deploying-your-prototype/README.md

```diff
@@ -0,0 +1,87 @@
+This guide explains how to deploy prototypes to MoJ's Heroku account.
+
+After following this guide your prototype will automatically deploy any changes that you push to GitHub.
+
+## Before you start
+
+You'll need to be a member of the [moj-design](https://dashboard.heroku.com/teams/moj-design/apps) Heroku team and have your prototype in [MoJ's GitHub account](https://github.com/ministryofjustice).
+
+If you don't have access to these, ask in the Slack channel [#moj-design-system](https://mojdt.slack.com/messages/CH5RUSB27) and whoever is managing support that day should be able to add you.
+
+<!-- If you don't know how to setup GitHub read the [version your prototype](#) guide. -->
+
+## Setting up Heroku
+
+We use pipelines in Heroku. Pipelines let you have multiple apps and keep things together in one place.
+
+1. go to the homepage for [moj-design](https://dashboard.heroku.com/teams/moj-design/apps) in Heroku
+2. click 'new' in the top right corner of the screen and select 'Create new pipeline'
+3. name the pipeline `{organisation-abbreviation}-{service-name}-prototype`
+4. leave the pipeline owner set to `moj-design`
+5. in the connect to GitHub section select `ministryofjustice` and type the name of your repo which is normally `{service-name}-prototype` and click search
```

Comment:
Append `{organisation-abbreviation}-` to the repo name (i.e., `{organisation-abbreviation}-{service-name}-prototype`). For example, the service I work on would be: "laa-crown-court-defence-prototype".